site2/website/scripts/replace.js

```diff
@@ -33,7 +33,7 @@ function connectorReleaseUrl(version) {
     if (version.includes('incubating')) {
         return `https://archive.apache.org/dist/incubator/pulsar/pulsar-${version}/apache-pulsar-io-connectors-${version}-bin.tar.gz`
     } else {
-        return `https://archive.apache.org/dist/pulsar/pulsar-${version}/apache-pulsar-io-connectors-${version}-bin.tar.gz`
+        return `https://archive.apache.org/dist/pulsar/pulsar-${version}/connectors/`
```

Comment:
2.3 is this url, 2.2.x url is the same.

Reply:
Seems this is for future release to use. Old site is already there.

Reply:
> Seems this is for future release to use. Old site is already there.
1.incubator version url is  https://archive.apache.org/dist/incubator/pulsar/pulsar-${version}/apache-pulsar-io-connectors-${version}-bin.tar.gz
2. 2.2.0 and 2.2.1 url is also https://archive.apache.org/dist/pulsar/pulsar-${version}/apache-pulsar-io-connectors-${version}-bin.tar.gz.
3. 2.3.0 is https://archive.apache.org/dist/pulsar/pulsar-${version}/connectors/

Reply:
> > Seems this is for future release to use. Old site is already there.
> > 1.incubator version url is  [https://archive.apache.org/dist/incubator/pulsar/pulsar-${version}/apache-pulsar-io-connectors-${version}-bin.tar.gz](https://archive.apache.org/dist/incubator/pulsar/pulsar-$%7Bversion%7D/apache-pulsar-io-connectors-$%7Bversion%7D-bin.tar.gz)
> 
> 1. 2.2.0 and 2.2.1 url is also [https://archive.apache.org/dist/pulsar/pulsar-${version}/apache-pulsar-io-connectors-${version}-bin.tar.gz](https://archive.apache.org/dist/pulsar/pulsar-$%7Bversion%7D/apache-pulsar-io-connectors-$%7Bversion%7D-bin.tar.gz).
> 2. 2.3.0 is [https://archive.apache.org/dist/pulsar/pulsar-${version}/connectors/](https://archive.apache.org/dist/pulsar/pulsar-$%7Bversion%7D/connectors/)

oh, sorry! I made a mistake.

Reply:
Thanks @ambition119  checked, this will run every build.
@liketic We should do the check  here,
only if (version > 2.3.0)  then use https://archive.apache.org/dist/pulsar/pulsar-${version}/connectors/

Reply:
@jiazhai Will update soon.